test/testfixture/environments.go

```diff
@@ -0,0 +1,47 @@
+package testfixture
+
+import (
+	"github.com/fabric8-services/fabric8-wit/path"
+	errs "github.com/pkg/errors"
+)
+
+// CreateWorkItemEnvironment returns a higher level recipe function that
+// contains a little bit more business logic compared to the plain recipe
+// functions that only set up dependencies according to database needs. In this
+// particular case, we create a root iteration and root area and make additional
+// areas be child of the root instances.
+func CreateWorkItemEnvironment() RecipeFunction {
```

Comment:
Is it an env to create WorkItems or is it to create an Env for WorkItems? I guess a little bit of both, but...

Reply:
It does not create WI itself but sets up the environment. 
`Create Environment For WI`

Reply:
@aslakknutsen @pranavgore09 feel free to rename as you wish.